src/ucp/wireup/wireup.c

```diff
@@ -370,26 +369,38 @@ ucp_wireup_init_lanes_by_request(ucp_worker_h worker, ucp_ep_h ep,
     return status;
 }
 
+#define UCP_WIREUP_MSG_CHECK(_msg, _ep, _msg_type) \
```

Comment:
can it be a function instead of macro?

Reply:
it was my first idea to use a function instead, but Clang static checker is unhappy and generates warnings that UCP WIREUP MSG handlers can access an EP which is `NULL`, i.e. `ucs_assert()`s in the function doesn't help

Reply:
we can suppress them, but then no reason to have the common macro for handlers

Reply:
i see, then let's comment this macro:
`/* Validate wireup message, implemented as a macro to prevent static checker warnings */`

Reply:
also let's move this macro to beginning of the file?

Reply:
> i see, then let's comment this macro:
> `/* Validate wireup message, implemented as a macro to prevent static checker warnings */`

makes sense to me. can force-push the changes? it is simple fix

Reply:
> makes sense to me. can force-push the changes? it is simple fix

yes, this is small PR

Reply:
done